region/maridia/outer/Main Street.json

```diff
@@ -3920,6 +4137,79 @@
       },
       "devNote": "This depends on how far the Morph tunnel is expanded, but this is both the minimum distance and the distance Map Rando has expanded."
     },
+    {
+      "link": [5, 5],
+      "name": "Sciser Farm",
+      "requires": [
+        "Morph",
+        {"resetRoom": {"nodes": [5]}},
+        {"or": [
+          {"and": [
+            "Gravity",
+            {"or": [
+              {"and": [
+                "ScrewAttack",
```

Comment:
springball speeds this up by 100f
And I actually see 450f on screw and pseudoscrew

And grapple is missing from these lists